### PR TITLE
travis: fix alpine builds

### PR DESCRIPTION
**Pull request by adrianreber**
With the latest version of the alpine container image it seems that alpine changed a few package names. This adapts the alpine container to solve the travis failures.

This also includes a small change to zdtm.py because the latest run of flake8 complains about having variables named 'l'.

### CODE REVIEW

**Comment by avagin** [comment]
I don't see this commit in the criu-dev branch. Can we cherry-pick proper commits from the criu-dev branch?

File: test/zdtm.py [hunk 1]
```diff
@@ -2077,11 +2077,11 @@ def grep_errors(fname):
     print_next = False
     before = []
     with open(fname, errors='replace') as fd:
-        for l in fd:
-            before.append(l)
+        for line in fd:
```
Comment (avagin) [inline test/zdtm.py:2080]: zdtm.py and Dockerfile.alpint changes should be in separate patches because these changes are independent.

**Comment by adrianreber** [comment]
> I don't see this commit in the criu-dev branch. Can we cherry-pick proper commits from the criu-dev branch?

I meant, that once this is merged on criu-dev it would be nice to have it on master to avoid broken travis runs.

**Comment by adrianreber** [comment]
Spilt into two commits.
